package/nao/package

```diff
@@ -2,17 +2,17 @@
 pkgname=nao
 pkgdesc="nao package manager"
 url=https://github.com/rmkit-dev/rmkit
-pkgver=0.0.1-14
-timestamp=2020-09-21T01:41+00:00
+pkgver=0.0.1-15
+timestamp=2020-09-27T01:41+00:00
 section=utils
 maintainer="raisjn <of.raisjn@gmail.com>"
 license=MIT
 depends=simple
 conflicts=rmkit
```

Comment:
Do we still need this since `rmkit` does not exist anymore and has never been moved to `stable`? Ditto for `simple`.

Reply:
people may have installed rmkit from testing, as most people are using testing (since nothing is in stable). will remove in some time from now (maybe a month or two)